WordPress/Tests/WP/CronIntervalUnitTest.inc

```diff
@@ -160,3 +160,45 @@ add_filter( 'cron_schedules', function ( $schedules ) {
 	];
 	return $schedules;
 } ); // Error: 8 min.
+
+// Correctly handle fully qualified WP time constants.
+class FQNConstants {
+	public function __construct() {
```

Comment:
```suggestion
	public function add_schedules() {
```

Change only because adding `add_*()` to class constructors is an anti-pattern that makes code less testable generally and I'd rather the fixtures didn't promote a bad practice.

Reply:
The constructor is used in some other places in this test case file as well but I'll update the new example.